### ISSUE TITLE
Bigger query?

### ISSUE THREAD

**Issue by sebkont** [TheBlewish/Automated-AI-Web-Researcher-Ollama#55]
Is there a way to type more in the query for context instead of very basic research query? Currently I can only type 5-6 sentences at best, before cmd window doesn't allow me to type any more, like there is a limit or something. Can something be easily changed to fix that?

PS: Unless it's not correct behavior and only occurs because i'm running on windows?

Since I assume it has nothing to do with token limits, just with code of the script

**Comment by TheBlewish** [comment]
![image](https://github.com/user-attachments/assets/b493f493-5378-4eab-8e5b-ee91f379de84)
As you can see it does appear to be a windows issue, is this with the windows branch of the program?
Or did you somehow get the main to work on windows?

**Comment by TheBlewish** [comment]
But yeah let me know, as you can see on Linux there isn't really a limit I did a long nonsense research query to demonstrate how it works on Linux, maybe provide a screenshot of what you mean if you could?
Also let me know if the branch for windows is the one that your using too!

**Comment by hafeezhmha** [comment]
@sebkont yeah please let us know if its the windows branch. I shall test this out too and will patch it up.

**Comment by leapingturtlefrog** [comment]
@sebkont The Windows version is the Windows branch here: [https://github.com/TheBlewish/Automated-AI-Web-Researcher-Ollama/tree/feature/windows-support](https://github.com/TheBlewish/Automated-AI-Web-Researcher-Ollama/tree/feature/windows-support)
